### DISCUSSION TITLE
Auth with Nextjs SSR doesn't work on mobile

### DISCUSSION THREAD

**Discussion by gabedealmeida** [supabase/supabase#3819] [Questions]
Auth with Nextjs SSR, as demonstrated by the Supabase official example https://github.com/supabase/supabase/tree/master/examples/nextjs-with-supabase-auth, works perfectly when using the desktop on every major browser. However, it doesn't work on any of the major browsers (Chrome, Firefox, Safari, or Edge) when using mobile. On mobile, `supabase.auth.api.getUserByCookie(req);` returns `null`. I'm wondering if this is a known issue and/or if anyone knows the fix to this problem. Thanks, team!

**Comment by kangmingtay** [comment]
hey @gabedealmeida, just checking, the `getUserByCookie` returns a promise object that contains the following fields (https://github.com/supabase/gotrue-js/blob/master/src/GoTrueApi.ts#L527-L531). Could you clarify which fields are being returned as `null` and if there is an error message being returned? Thanks!
